paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsViewModel.kt

```diff
@@ -30,11 +32,14 @@ class SavedPaymentMethodsViewModel(
     )
     val state: StateFlow<SavedPaymentMethodsViewState> = _state
 
+    private var customerId: String = ""
```

Comment:
Hmm, should we pass this through to the `setupIntentClientSecretProvider` from the adapter? It sucks to have to hold this in merchant code.